web/src/lib/components/shared-components/settings/setting-input-field.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { render } from '@testing-library/svelte';
+import userEvent from '@testing-library/user-event';
+// @ts-expect-error import works
```

Comment:
This should be unnecessary, I pulled the branch locally and after an npm install the error went away

Reply:
Right, I see that now, why that doesn't error in the IDE that also runs typescript against the file, I have no idea...

Reply:
The annoying part for me is that the ts-expect-error line errors because my IDE doesn't believe there is actually an error there :laughing:

Reply:
I have only seen and run into issues when using `<script lang="ts" context="module">`. Personally, I just prefer to externalize it into a dedicated ts file.